Sources/Plasma/CoreLib/hsStream.cpp

```diff
@@ -819,6 +810,39 @@ uint32_t hsWriteOnlyStream::Write(uint32_t byteCount, const void* buffer)
     return byteCount;
 }
 
+void hsWriteOnlyStream::Skip(uint32_t deltaByteCount)
+{
+    fPosition += deltaByteCount;
+    fData += deltaByteCount;
+    if (fData > fStop) {
+        hsThrow("Skip went past end of stream");
+    }
+}
+
+void hsWriteOnlyStream::Rewind()
+{
+    fPosition = 0;
+    fData = fStart;
+}
+
+void hsWriteOnlyStream::FastFwd()
+{
+    fPosition = GetEOF();
+    fData = fStop;
+}
+
+void hsWriteOnlyStream::Truncate()
+{
+    hsThrow("can't write to a readonly stream");
```

Comment:
Oops. This was copied from `hsReadOnlyStream`, which was the superclass before. `hsWriteOnlyStream` really cannot be truncated though, because it's based on a fixed-size memory buffer. Will update the message.